### PR TITLE
chore(demo-playwright): test for `tui-error`

### PR DESCRIPTION
**Pull request by AtharvChandratre**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?
This PR adds an e2e test (with playwright) for `tui-error`

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-playwright/tests/core/error/tui-error.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import {expect, test} from '@playwright/test';
+
+test.describe(`TuiError`, () => {
+    test(`No extra space between content`, async ({page}) => {
+        await tuiGoto(page, `components/error#base`);
+        const example = page.locator(
+            `tui-doc-example[heading="Basic"] [automation-id="tui-doc-example"]`,
+        );
+
+        await example.locator(`[automation-id="tui-toggle__checkbox"]`).click();
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/error/tui-error.spec.ts:11]: Let's simplify selector

```suggestion
        await example.locator(`tui-toggle input[type="checkbox"]`).click();
```

I hope there is no need to reveal the complex `tui-toggle`'s structure 🤔

File: projects/demo-playwright/tests/core/error/tui-error.spec.ts
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/error/tui-error.spec.ts:1]: `tui-error.spec.ts` => `error.spec.ts`

File: projects/demo-playwright/tests/core/error/tui-error.spec.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/core/error/tui-error.spec.ts:11]: The same for `tui-error`

Comment (nsbarsukov) [reply projects/demo-playwright/tests/core/error/tui-error.spec.ts:11]:
```ts
const checkbox = example.locator(`tui-toggle input[type="checkbox"]`);
const error = example.locator(`tui-error`);

await expect(error).not.toBeVisible();
await checkbox.click();

await expect(checkbox).toBeChecked();
await expect(error).toBeVisible();
await expect(error).toHaveText(`An error`);

await expect(example).toHaveScreenshot(`01-error.png`);
```

**Comment by AtharvChandratre** [comment]
@nsbarsukov thank you for the comments, have addressed them

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**